README.md

```diff
@@ -58,9 +58,13 @@ associated workflow.
 Link triggers are _unique to each installed version of your app_. This means
 that Shortcut URLs will be different across each workspace, as well as between
 [locally run](#running-your-project-locally) and
-[deployed apps](#deploying-your-app). When creating a trigger, you must select
-the Workspace that you'd like to create the trigger in. Each Workspace has a
-development version (denoted by `(dev)`), as well as a deployed version.
+[deployed apps](#deploying-your-app).
+
+When creating a trigger, you must select the workspace and environment that
```

Comment:
Looping you in on this change @hello-ashleyintech, as we'll want to make the same change across all repositories at some point. 

Jesse was kind enough to open up a PR that reflects the new changes to the trigger creation flow. It's subtle, but we've added an environment selector. I edited the submission a bit to reduce it to only what's necessary, but wanted to ensure I wasn't missing or overlooking anything.